codes/swift/chapter_computational_complexity/time_complexity.swift

```diff
@@ -88,7 +88,7 @@ func expRecur(n: Int) -> Int {
 }
 
 /* 对数阶（循环实现） */
-func logarithmic(n: Int) -> Int {
+func logarithmic(n: Double) -> Int {
```

Comment:
Could we revert back `n` as Int for the consistency with other languages.

Reply:
Use `Int` will cause a `count` error because the `/` operator.

Reply:
Do you mean compile error or numeric error? 
I think the latter is fine because the complexity is still correct. 

- For n is even, O(n)
- For n is odd, O(n - 0.5) = O(n)